src/app/analyze/analyze-overview/analyze-overview.component.scss

```diff
@@ -1,55 +1,79 @@
-.team {
-  .img-icon{
-    width: 25px;
-    height: 25px;
-  }
-  .member {
-    height: 35px;
-  }
-}
-
-.number {
-  font-size: 96px;
-}
-
-.panel-300 {
-  .panel-body {
-    height: 300px;
-  }
-}
-
-.panel-title {
-  min-height: 30px;
-}
+@import '../../../assets/stylesheets/base';
+@import '../../../assets/stylesheets/shared/layout';
 
-.panel-140 {
-  .panel-body {
-    height: 140px;
+.analyze-overview-wrapper {
+  .space-overview-wrapper {
+    padding-bottom: 0;
+    @extend .flex-container;
+    > div {
+      aside {
+        padding: 0 rem(20);
+        height: 100%;
+        @extend .flex-container;
+        .card-pf {
+          @extend .flex-container;
+          flex: 1;
+          -webkit-flex: 1;
+          min-height: rem(288);
+          .badge {
+            margin-top: rem(-4);
+          }
+          .analyze-overview-add-items {
+            @extend .flex-container;
+            @extend .in-column-direction;
+            align-items: center;
+            .pficon-add-circle-o {
+              font-size: rem(50);
+              color: $color-pf-black-400;
+              margin-bottom: rem(5);
+            }
+            .card-pf-utilization-lg-p {
+              font-size: rem(25);
+              color: $color-pf-black-800;
+              font-weight: 300;
+              margin: rem(10) 0;
+            }
+          }
+          .analyze-overview-items-detail {
+            @extend .flex-container;
+            @extend .in-column-direction;
+            @extend .flex-grow-1;
+            .card-pf-body {
+              flex: auto;
+              -webkit-flex: auto;
+              ul {
+                list-style: none;
+                padding: 0;
+                li {
+                  padding-bottom: rem(10);
+                }
+              }
+            }
+            .card-pf-footer {
```

Comment:
this should not be needed when the blank state component is used. Default PatternFly link behavior should be used.